src/Frontend/Core.hs

```diff
@@ -412,7 +412,7 @@ form formHandler = getH :<|> postH
     processor2 page result = absoluteUriPath . relPath . redirectOf page <$> processor result
 
 
--- * frames
+-- * frame creation
```

Comment:
This commit is great, I am glad I recovered the diff even though you did a good job hiding it.  (Spoiiler: `pageFrame` now takes a frame and is much easier to call from the `ToHtml` instance and elsewhere.)